src/wsi/meson.build

```diff
@@ -26,6 +31,9 @@ elif dxvk_wsi == 'sdl2'
 elif dxvk_wsi == 'glfw'
   wsi_src  = wsi_common_src + wsi_glfw_src
   wsi_deps = [ dep_displayinfo, lib_glfw ]
+elif dxvk_wsi == 'none'
+  wsi_src  = wsi_common_src + wsi_none_src
+  wsi_deps = [ dep_displayinfo ]
```

Comment:
I suppose `dep_displayinfo` and `wsi_common_src` (`wsi_edid.cpp`) are not required for this.
+1 TODO if yes

Reply:
ah, thanks!

Reply:
I added `wdi_edid_none` to replace `parseColorimetryInfo`.